bot_controller/src/routes.rs

```diff
@@ -154,12 +150,19 @@ pub async fn start_bot(
             // }
         }
     }
-    let (game_host, game_port) = (get_game_host(PREFIX), get_game_port(PREFIX));
 
-    let temp_game_host = format!("{game_host}:{game_port}");
+    let game_host =
+        std::env::var(format!("ACBOT_GAME_HOST")).unwrap_or_else(|_| "127.0.0.1".into());
```

Comment:
If you have the time, it might be ideal to extract this and other related configuration logic out into a configuration file/module, just so it's clear what a config looks like and what the defaults are.